tmt-app/src/Styling/Home.css

```diff
@@ -1,71 +1,34 @@
 /*
- Subheader styling
+ Styling For home path arrow SVG
 */
-.Home_Path_Wrapper{
-    width: 100%;
-    display: flex;
-    align-items: center;
-}
-.Home_Path_Wrapper img{
+#Home_Path_Arrow{
```

Comment:
Missed this earlier and we merged already, but for next time, want to point out that ID selectors are typically bad practice just due to the weight that the ID has on the selector

https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Can read about CSS Specificity above, really good thing to understand to minimize css headaches down the road